packages/libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -67,7 +62,6 @@ type DiscoveryResponse<Response> = {
 export type DiscoveryProviderConfig = {
   whitelist?: Set<string>
   blacklist?: Set<string>
-  userStateManager: UserStateManager
```

Comment:
can we delete this file entirely?

https://github.com/AudiusProject/audius-protocol/blob/70a05001595234f322bd0488d5f2b0aa9be931e7/packages/libs/src/userStateManager.ts#L1